onebusaway-android/src/main/java/org/onebusaway/android/map/MapModeController.java

```diff
@@ -66,12 +69,20 @@
 
         void showStops(List<ObaStop> stops, ObaReferences refs);
 
+        void showBikeStations(List<BikeRentalStation> bikeStations);
```

Comment:
On the topic of separating the code/logic for multiple layers - I think this will change to something like:

`void showLayer(Layer layer);`

And, each `*Overlay` class would implement a new `Layer` interface.